frontend/src/styles.css

```
@@ -1,6 +1,6 @@
 @import 'tailwindcss';
 
-@config '../tailwind.config.ts';
+@custom-variant dark (&:where(.dark, .dark *));
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)